packages/lumx-react/src/components/tabs/TabList.test.tsx

```diff
@@ -96,36 +97,21 @@ describe(`<${Tabs.displayName}>`, () => {
 
     // 3. Test events.
     describe('Events', () => {
-        const onTabClick: jest.Mock = jest.fn();
+        const onChange: jest.Mock = jest.fn();
 
         beforeEach(() => {
-            onTabClick.mockClear();
+            onChange.mockClear();
         });
 
-        it('should trigger `onTabClick` when a child tab is clicked', () => {
-            const { wrapper } = setup({ onTabClick }, false);
-            const firstTab = wrapper.find('Tab[index=1]');
+        it('should trigger `onChange` when a child tab is clicked', () => {
+            const { wrapper } = setup({ onChange }, false);
+            const firstTab = wrapper.find('Tab[id="tab-1"]');
 
             firstTab.simulate('click');
-            expect(onTabClick).toHaveBeenCalledWith({ event: jasmine.any(Object), index: 1 });
-        });
-    });
-
-    // 4. Test conditions (i.e. things that display or not in the UI based on props).
-    describe('Conditions', () => {
-        it('should fail when no `Tab` children is given', () => {
-            expect(() => {
-                // We know that children must be given to <Tabs>, but for the test, ignore it.
-                setup({ children: null });
-            }).toThrowErrorMatchingSnapshot();
+            expect(onChange).toHaveBeenCalledWith('tab-1');
         });
     });
```

Comment:
Following the "isDisabled" modification, here's a new test to check that it works properly. :) 

```suggestion

        it('should not trigger `onChange` when a disabled child tab is clicked', () => {
            const tabs = [
                <Tab id="tab-0" label="Tab 0" aria-controls="tab-panel-0" />,
                <Tab id="tab-1" label="Tab 1" aria-controls="tab-panel-1" isDisabled />,
            ];
            const { wrapper } = setup({ onChange, children: tabs }, false);
            const secondTab = wrapper.find('Tab[id="tab-1"]');

            secondTab.simulate('click');
            expect(onChange).not.toHaveBeenCalled();
        });
    });
```